docs/pages/integrations/apache-shardingsphere-operator-charts.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Apache Shardingsphere Operator Charts
+permalink: integrations/apache-shardingsphere-operator-charts
+type: installation
+category: integrations
+display-title: "true"
```

Comment:
We'll want to have new frontmatter values, like:

```
model: 
model-category:
model-subcategory: (in the future)
```

Reply:
@leecalcote sure will work on it :)

Reply:
@Lee do i need to change the Category to Model right ??

Reply:
Thank you. In the meantime.... the meshery.io site is pointing to "undefined" links. 😬